src/rendezvous/MSC3906Rendezvous.ts

```diff
@@ -64,18 +94,23 @@ export class MSC3906Rendezvous {
     private newDeviceId?: string;
     private newDeviceKey?: string;
     private ourIntent: RendezvousIntent = RendezvousIntent.RECIPROCATE_LOGIN_ON_EXISTING_DEVICE;
+    private v1FallbackEnabled: boolean;
     private _code?: string;
 
     /**
      * @param channel - The secure channel used for communication
      * @param client - The Matrix client in used on the device already logged in
      * @param onFailure - Callback for when the rendezvous fails
+     * @param flow - The flow to use. Defaults to MSC3906 v1 for backwards compatibility.
```

Comment:
please could we have some more words explaining what a "flow" means and how I, as a user of this API, should pick one?